detail.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from django.http import HttpResponseRedirect
+from django_mako_plus import view_function, jscontext, RedirectException
+from catalog import models as cmod
+from formlib import Formless
+import math
+
```

Comment:
I'd think about deleting import statements that you don't use in this file.

Reply:
Yes, I will do that!